cli/config/config.go

```diff
@@ -0,0 +1,112 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/nspcc-dev/neo-go/cli/cmdargs"
+	"github.com/nspcc-dev/neo-go/pkg/config"
+	"github.com/urfave/cli"
+	"gopkg.in/yaml.v3"
+)
+
+// NewCommands returns 'config' command.
+func NewCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:  "config",
+			Usage: "Configuration management",
+			Subcommands: []cli.Command{
+				{
+					Name:      "generate",
+					Usage:     "Generate configuration file",
+					UsageText: "neo-go config generate [-flags]",
+					Action:    configGenerate,
+					Flags: []cli.Flag{
+						cli.BoolFlag{Name: "privnet", Usage: "private network configuration"},
+						cli.BoolFlag{Name: "mainnet", Usage: "mainnet network configuration"},
+						cli.BoolFlag{Name: "testnet", Usage: "testnet network configuration"},
+						cli.BoolFlag{Name: "unit_testnet", Usage: "unit test network configuration"},
+						cli.BoolFlag{Name: "mainnet_neofs", Usage: "mainnet NeoFS network configuration"},
+						cli.BoolFlag{Name: "testnet_neofs", Usage: "testnet NeoFS network configuration"},
+						cli.BoolFlag{Name: "docker", Usage: "Docker configuration"},
+						cli.BoolFlag{Name: "all", Usage: "all networks configurations"},
+					},
+				},
+			},
+		},
+	}
+}
+
+func configGenerate(ctx *cli.Context) error {
+	if err := cmdargs.EnsureNone(ctx); err != nil {
+		return err
+	}
+
+	var filesToGenerate map[string]config.Config
+
+	if ctx.Bool("all") {
+		filesToGenerate = map[string]config.Config{
+			"protocol.mainnet.yml":               MainNetConfig,
```

Comment:
Here and below it's `Mainnet`, `MainnetNeoFS` and etc. We're already in the `config` package.